src/contracts/core/DelegationManager.sol

```diff
@@ -360,6 +362,35 @@ contract DelegationManager is
         });
     }
 
+    /**
+     *
+     *                         BACKWARDS COMPATIBLE LEGACY FUNCTIONS
+     *                         TO BE DEPRECATED IN FUTURE
+     *
```

Comment:
I feel like we may want a non-reverting `getCompletableTimestamp` function? Wdyt